File: py/exerc001.py
#**********************************************************************
# FILE NAME   : EXERC001                                              *
# DATE        : 2025-03-26                                            *
# AUTHOR      : FABIO MARQUES (FMARQUES@FMARQUES.ETI.BR)              *
# DATA CENTER : COMPANY.EDUC360                                       *
# PURPOSE     : CALCULATE THE RECTANGLE AREA                          *
#**********************************************************************

# Variable declaration
base   = 0
altura = 0
area   = 0

# Reading the rectangle base value
base   = input("Informe o valor da base:   ")
# Reading the rectangle height value
altura = input("Informe o valor da altura: ")

# Rectangle area calculation
area = float(base) * float(altura)

# Display the calculated area
print("\nA área do triangulo retângulo é ", area)

File: js/exerc001.js
/**********************************************************************
* FILE NAME   : EXERC001                                              *
* DATE        : 2025-03-29                                            *
* AUTHOR      : FABIO MARQUES (FMARQUES@FMARQUES.ETI.BR)              *
* DATA CENTER : COMPANY.EDUC360                                       *
* PURPOSE     : CALCULATE THE RECTANGLE AREA                          *
**********************************************************************/

// Variable declaration
let base   = 0
let altura = 0
let area   = 0

// Reading the rectangle base and height values
base   = prompt("Informe a base") 
altura = prompt("Informe a altura")

// Rectangle area calculation
area   = base * altura

// Display the calculated area
console.log("A área do retângulo é " + area)